go.mod

```diff
@@ -286,3 +288,11 @@ require (
 	gopkg.in/yaml.v3 v3.0.1 // indirect
 	kernel.org/pub/linux/libs/security/libcap/psx v1.2.66 // indirect
 )
+
+replace github.com/hashicorp/raft => /Users/swenson/projects/raft
+
+replace github.com/hashicorp/raft-boltdb/v2 => /Users/swenson/projects/raft-boltdb/v2
+
+replace github.com/hashicorp/memberlist => /Users/swenson/projects/memberlist
+
+replace github.com/hashicorp/serf => /Users/swenson/projects/serf
```

Comment:
woops! I think this is causing all the current CI failures

Reply:
Yep, there is a parameter being added in these (`MsgpackUseNewTimeFormat`) to help with compatibility, so this PR's build won't work until these are merged anyway.